src/main/java/core/basesyntax/service/service/WriteService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.service;
```

Comment:
```suggestion
package core.basesyntax.service;
```
You don't need to have service in service. Let’s  remove the subpackage